app/src/ui/app.tsx

```diff
@@ -358,6 +360,21 @@ export class App extends React.Component<IAppProps, IAppState> {
     })
   }
 
+  private goToSummary() {
```

Comment:
This method duplicates `showChanges` but also emits an event when `changeRepositorySection` is resolved. It could be made DRY if `showChanges` is refactored to return `changeRepositorySection`'s promise.